components/Badge/badge.css

```diff
@@ -0,0 +1,50 @@
+.badge {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-end;
+  gap: 1rem;
+  justify-content: center;
+}
+.badge > div {
+  position: relative;
+}
+.num {
+  position: absolute;
+  border: var(--color-white) solid 2px;
```

Comment:
You could also just add `--white` for the variable name.

Reply:
initially, my plan was to have normal text white colour and background with different shade's of white but not added that so --color-bg-white something like this.